code/modules/surgery/bodyparts/robot_bodyparts.dm

```diff
@@ -158,3 +158,34 @@
 		flash2.forceMove(user.loc)
 		flash2 = null
 	..()
+
+
+
+
+/obj/item/bodypart/l_arm/robot/surplus
+	name = "surplus prosthetic left arm"
+	desc = "A skeletal, robotic limb. Outdated and fragile, but it's still better than nothing."
+	icon = 'icons/mob/augments.dmi'
+	icon_state = "surplus_l_arm"
+	max_damage = 20
+
+/obj/item/bodypart/r_arm/robot/surplus
+	name = "surplus prosthetic right arm"
+	desc = "A skeletal, robotic limb. Outdated and fragile, but it's still better than nothing."
+	icon = 'icons/mob/augments.dmi'
+	icon_state = "surplus_r_arm"
+	max_damage = 20
+
+/obj/item/bodypart/l_leg/robot/surplus
+	name = "surplus prosthetic leg"
+	desc = "A skeletal, robotic limb. Outdated and fragile, but it's still better than nothing."
+	icon = 'icons/mob/augments.dmi'
+	icon_state = "surplus_l_leg"
+	max_damage = 20
+
+/obj/item/bodypart/r_leg/robot/surplus
+	name = "surplus prosthetic leg"
+	desc = "A skeletal, robotic limb. Outdated and fragile, but it's still better than nothing."
+	icon = 'icons/mob/augments.dmi'
+	icon_state = "surplus_r_leg"
+	max_damage = 20
```

Comment:
These should be craftable, somehow. Maybe order a crate of surplus limbs from cargo?